### PR TITLE
[Minor-Fix][362] Several galleries per page.

### PR DESCRIPTION
**Pull request by ivan-berezhnov**
GitHub Issue #362
Jira issue https://propeople-us.atlassian.net/browse/OYPD-454
Drupal.org issue: https://www.drupal.org/node/2871505

Steps:
- [x] login as admin
- [x] go to any page
- [x] add two or more galleries. to the page and to one section
- [x] click on '<' or '>' in the second gallery
- [x] verify slide change for the second gallery

### CODE REVIEW

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1271 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1271 |
| Installation process | http://install.openy-dev.ffwua.com/build1271/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1271/

**Comment by gundevel** [comment]
Behat test results file URL: http://ci.openymca.org/build1271/profiles/contrib/openy/build/reports/behat

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1272 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1272 |
| Installation process | http://install.openy-dev.ffwua.com/build1272/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1272/

**Comment by gundevel** [comment]
Behat test results file URL: http://ci.openymca.org/build1272/profiles/contrib/openy/build/reports/behat

File: modules/openy_features/openy_prgf/modules/openy_prgf_gallery/openy_prgf_gallery.module [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+/**
+ * @file
+ * OpenY Paragraph Gallery module file.
+ */
+
+/**
+ * Implements hook_preprocess_node().
+ */
+function openy_prgf_gallery_preprocess_node(&$variables) {
+
```
Comment (alexschedrov) [inline modules/openy_features/openy_prgf/modules/openy_prgf_gallery/openy_prgf_gallery.module]: Please remove empty line.

File: modules/openy_features/openy_prgf/modules/openy_prgf_gallery/openy_prgf_gallery.libraries.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+gallery_alert:
+  version: VERSION
+  js:
+    js/openy_prgf_gallery_alert.js: {}
+  dependencies:
+    - core/jquery
```
Comment (alexschedrov) [inline modules/openy_features/openy_prgf/modules/openy_prgf_gallery/openy_prgf_gallery.libraries.yml]: Please add empty endline.

File: modules/openy_features/openy_prgf/modules/openy_prgf_gallery/js/openy_prgf_gallery_alert.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+        $(this).attr('id', 'carousel' + i);
+        $(this).find('.carousel-control').each(function () {
+          $(this).attr('href', '#carousel' + i);
+        });
+        i++;
+      });
+    }
+  };
+
+})(jQuery);
```
Comment (alexschedrov) [inline modules/openy_features/openy_prgf/modules/openy_prgf_gallery/js/openy_prgf_gallery_alert.js]: Please add empty endline.

**Review by alexschedrov — CHANGES_REQUESTED** [review]

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1273 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1273 |
| Installation process | http://install.openy-dev.ffwua.com/build1273/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1273/

**Comment by gundevel** [comment]
Behat test results file URL: http://ci.openymca.org/build1273/profiles/contrib/openy/build/reports/behat

File: modules/openy_features/openy_prgf/modules/openy_prgf_gallery/openy_prgf_gallery.module [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+ * @file
+ * OpenY Paragraph Gallery module file.
+ */
+
+/**
+ * Implements hook_preprocess_node().
+ */
+function openy_prgf_gallery_preprocess_node(&$variables) {
+  // Attach openy_prgf_gallery_alert.js.
+  $variables['#attached']['library'][] = 'openy_prgf_gallery/gallery_alert';
```
Comment (alexschedrov) [inline modules/openy_features/openy_prgf/modules/openy_prgf_gallery/openy_prgf_gallery.module]: What if node doesn't have this paragraph at all? This function will attache library to every single node. Let's find a bettew way of attaching libraries.

**Review by alexschedrov — CHANGES_REQUESTED** [review]

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1296 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1296 |
| Installation process | http://install.openy-dev.ffwua.com/build1296/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1296/

**Comment by gundevel** [comment]
Behat test results file URL: http://ci.openymca.org/build1296/profiles/contrib/openy/build/reports/behat

**Comment by ivan-berezhnov** [comment]
retest this please

File: modules/openy_features/openy_prgf/modules/openy_prgf_gallery/openy_prgf_gallery.libraries.yml (hunk 2 above)
Comment (ivan-berezhnov) [reply modules/openy_features/openy_prgf/modules/openy_prgf_gallery/openy_prgf_gallery.libraries.yml]: Added empty end line.

File: modules/openy_features/openy_prgf/modules/openy_prgf_gallery/js/openy_prgf_gallery_alert.js (hunk 3 above)
Comment (ivan-berezhnov) [reply modules/openy_features/openy_prgf/modules/openy_prgf_gallery/js/openy_prgf_gallery_alert.js]: Added empty end line.

**Comment by bjwilkins** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1298 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1298 |
| Installation process | http://install.openy-dev.ffwua.com/build1298/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1298/

**Comment by bjwilkins** [comment]
Behat test results file URL: http://ci.openymca.org/build1298/profiles/contrib/openy/build/reports/behat

**Review by alexschedrov — APPROVED** [review]
Approved.

**Pull request merged**